src/application/requestExecutor/RequestExecutorFactory.ts

```diff
@@ -0,0 +1,19 @@
+import InvalidContentTypeError from '@core/domain/common/utils/error/InvalidContentTypeError';
+import RequestExecutor from '@core/domain/common/port/RequestExecutorPort';
+import UrlRequestExecutor from './UrlRequestExecutor';
+
+class RequestExecutorFactory  {
```

Comment:
1. Is it hooked up anywhere or is this still WIP?
1. In my opinion this RequestExecutor-related logic belongs in domain

Reply:
@1 WIP. I will click "request review" once it's ready, though you are welcome to have a look every now and then if you'd like to ;)
@2 Yeah, I couldn't decide - I felt there is some sense to both locations. I will move it to domain then.